stable/cloudzero-cloudwatch-metrics/templates/configmap.yaml

```diff
@@ -14,7 +14,7 @@ data:
         "metrics_collected": {
           "kubernetes": {
             "cluster_name": "{{ .Values.clusterName }}",
-            "metrics_collection_interval": 120
+            "metrics_collection_interval": {{ .Values.metricsCollectionInterval }}
```

Comment:
>we probably wouldn't want to allow a user to use a value here less than 60

Hmm, that's exactly what I need though 😅 I have cases where pods could exist for less than a minute and as a result the agent may miss and not collect any metrics for it.

Overall, I like the idea of having some kind of validation, but do you think it is better to have it as an allowlist of value instead of any integer being allowed? And would you be OK with values less than 60 seconds? Is AWS CloudWatch Logs cost the only reason for avoiding values less than 60 seconds?

Reply:
that's a fair point. allowing a lower integer bound sounds good to me. the only other reason to avoid low values is that we wouldn't want to send an overwhelming number of requests to the kube api, which could have adverse effects on the cluster. I think that's probably unlikely, and I can't imagine that any user would think to set a value so low as to really cause issues. 
how low are you intending to set the interval?

Reply:
Yes, this makes sense. My idea was to set it to 15 seconds, however, if you think it might be too low, maybe 30 seconds should be fine as well, basically half of what we currently have. Overall, I don't think I will ever get 100% accuracy, but a good approximation is what I am looking for. For the allowed values, do you think I could set the list to `[15, 30, 45, 60, 120]`? This will also give users options to try out and see what works for them.

Reply:
that list sounds good to me 👍

Reply:
this looks great, thanks! one additional note here is that sub-60 second intervals would result in the [high resolution metrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/publishingMetrics.html#high-resolution-metrics), which will incur additional cost. 

I do want to check that high resolution metrics don't break anything in our ingestion pipeline, so I'm going to test that out. as long as it doesn't break anything I'll throw an approval on here before EOD